### PR TITLE
Fix show more show less link collapse

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue [hunk 1]
```diff
@@ -161,7 +161,7 @@ export default defineComponent({
     const space = inject<Ref<SpaceResource>>('space')
     const resource = inject<Ref<Resource>>('resource')
 
-    const linkListCollapsed = !store.getters.configuration.options.sidebar.shares.showAllOnLoad
+    const linkListCollapsed = ref(!store.getters.configuration.options.sidebar.shares.showAllOnLoad)
```
Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue:164]: shouldn't this be a computed?!

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/35102/12/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/35102/tracing/space-members-can-be-invited-with-an-expiration-date-alice-2023-4-25-09-11-55.zip 


</p></details>
:boom: The e2e-ocis tests pipeline failed. The build has been cancelled.

File: packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue (hunk 1 above)
Comment (JanAckermann) [reply packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue]: reworked =)

**Review by kulmann — APPROVED** [review]

**Review by JammingBen — APPROVED** [review]

**Comment by kulmann** [comment]
PR needs a rebase :-)

**Pull request merged**